x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/assignment/AssignmentStats.java

```diff
@@ -269,8 +285,13 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 builder.field("inference_count", inferenceCount);
             }
             // avoid reporting the average time as 0 if count < 1
-            if (avgInferenceTime != null && (inferenceCount != null && inferenceCount > 0)) {
-                builder.field("average_inference_time_ms", avgInferenceTime);
+            if (inferenceCount != null && inferenceCount > 0) {
+                if (avgInferenceTime != null) {
+                    builder.field("average_inference_time_ms", avgInferenceTime);
+                }
+                if (avgResponseTime != null) {
+                    builder.field("average_response_time_ms", avgResponseTime);
```

Comment:
This field name doesn't correspond to the name in the docs change, nor to the name used in `PyTorchResultProcessor.java`.

Reply:
👍 I left that change on disk